### PR TITLE
[Doc][Improve] Support chinese for sinks

### PR DESCRIPTION
**Pull request by corgy-w**
### Purpose of this pull request

Translate multiple sink doc files

### Does this PR introduce any user-facing change?
no

### How was this patch tested?
mvn spotless:apply

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by corgy-w** [comment]
@EricJoy2048 @zhilinli123  @Hisoka-X Sorry, due to my inexperience, the PR I originally submitted was contaminated by me, so I closed the previous PR, please help me and take a look at it. The previous review has been changed and the translation of `common-options.md` has been completed. TKS!! :-)

**Comment by zhilinli123** [comment]
thanks ping. If you can, please translate Todo first, and then we can review it together for better results

**Comment by corgy-w** [comment]
> thanks ping. If you can, please translate Todo first, and then we can review it together for better results

File: docs/zh/connector-v2/sink/Elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+指定信任存储的密钥密码
+
+### common options
+
+Sink插件常用参数，请参考 [Sink常用选项](common-options.md) 了解详情
+
+### schema_save_mode
+
+同步任务开启前，针对目标侧已有的表面结构选择不同的处理方案
```
Comment (EricJoy2048) [inline docs/zh/connector-v2/sink/Elasticsearch.md]:
```suggestion
同步任务开启前，针对目标侧已有的表结构选择不同的处理方案
```

Hi, @corgy-w Thanks for your contribution. Chinese document is very important for us. I checked the content in this PR and found that many parts were only translated using translation tools, and the actual description is not very accurate, making it difficult for users to understand the true meaning they want to express. Can we make some modifications to make them look more reasonable?

Comment (corgy-w) [reply docs/zh/connector-v2/sink/Elasticsearch.md]: I'll review them all and make some tweaks in spare time . Thks you reminding :>

File: docs/zh/connector-v2/formats/canal-json.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  "table": "products",
+  "ts": 1589373560798,
+  "type": "UPDATE"
+}
+```
+
+注：请参考 Canal 文档以了解每个字段的含义
+
+MySQL 的 products 表有 4 列（id、name、description 和 weight）
+上述 JSON 消息是产品表的一个更新变更事件，其中 id = 111 的行的 weight 值从 5.18 变为 5.15
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/canal-json.md]:
```suggestion
上述 JSON 消息是产品表的一个更新变更事件，其中 id = 111 的行的 weight 值从 5.15 变为 5.18
```
Make the document description conform to the example

File: docs/zh/connector-v2/formats/canal-json.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    "description": 12,
+    "weight": 7
+  },
+  "table": "products",
+  "ts": 1589373560798,
+  "type": "UPDATE"
+}
+```
+
+注：请参考 Canal 文档以了解每个字段的含义
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/canal-json.md]:
```suggestion
注：请参考 [Canal 文档](https://github.com/alibaba/canal/wiki) 以了解每个字段的含义
```
Supplementary Canal document link

File: docs/zh/connector-v2/formats/canal-json.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+序列化模式、反序列化模式
+
+Canal是一款CDC（变更数据捕获）工具，能够实时捕获MySQL的数据变化并将其流式传输到其他系统中。Canal为变更日志提供了一种统一的格式，并支持使用 JSON 和 protobuf（Canal默认使用protobuf）进行消息的序列化
+
+SeaTunnel 能够解析 Canal 的 JSON 消息，并将其转化为 INSERT/UPDATE/DELETE 消息，进而输入到 SeaTunnel 系统中。这个特性在很多场景下都显得非常有用，例如:
+
+        将增量数据从数据库同步到其他系统
+        审计日志
+        数据库的实时物化视图
+        临时连接改变数据库表的历史记录等
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/canal-json.md]:
```suggestion
        关联维度数据库的变更历史，等等。
```
This refers to the translation of the [Flink document](https://nightlies.apache.org/flink/flink-docs-master/zh/docs/connectors/table/formats/canal/)

File: docs/zh/connector-v2/formats/avro.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
+# Avro 格式
+
+Avro是一种流行数据传输格式。现在seatunnel在kafka连接器中支持Avro格式
+
+# 怎样用
+
+## Kafka 使用示例
+
+- 使用 avro 格式从模拟数据源和接收器生成数据到 kafka 的示例
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/avro.md]:
```suggestion
- 使用 Avro 格式从模拟数据源生成数据并将其汇聚到 Kafka 的示例。
```

File: docs/zh/connector-v2/formats/debezium-json.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+		"thread": 2,
+		"query": null
+	},
+	"op": "u",
+	"ts_ms": 1589362330904,
+	"transaction": null
+}
+```
+
+注：请参考 Debezium 文档以了解每个字段的含义
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/debezium-json.md]:
```suggestion
注：请参考 [Debezium 文档](https://debezium.io/documentation/reference/2.5/connectors/mysql.html#mysql-events) 以了解每个字段的含义
```

File: docs/zh/connector-v2/formats/debezium-json.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
+# Debezium 格式
+
+变更数据捕获格式:
+序列化模式、反序列化模式
+
+Debezium 是一套分布式服务，用于捕获数据库中的变化，以便您的应用程序可以看到这些变化并对其做出响应。Debezium 在变更事件流中记录每个数据库表中的所有行级变化，应用程序只需读取这些流，就可以按照它们发生的顺序看到变更事件
+
+SeaTunnel 支持将 Debezium JSON 消息解释为 INSERT/UPDATE/DELETE 消息并导入到 seatunnel 系统中。在许多情况下，利用这个特性是非常有用的，例如:
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/debezium-json.md]:
```suggestion
SeaTunnel 支持将 Debezium JSON 消息解析为 INSERT/UPDATE/DELETE 消息并导入到 seatunnel 系统中。在许多情况下，利用这个特性是非常有用的，例如:
```

File: docs/zh/connector-v2/formats/debezium-json.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+	"op": "u",
+	"ts_ms": 1589362330904,
+	"transaction": null
+}
+```
+
+注：请参考 Debezium 文档以了解每个字段的含义
+
+MySQL 的 products 表有 4 列（id、name、description 和 weight）
+上述 JSON 消息是产品表的一个更新变更事件，其中 id = 111 的行的 weight 值从 5.18 变为 5.15
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/debezium-json.md]:
```suggestion
上述 JSON 消息是产品表的一个更新变更事件，其中 id = 111 的行的 weight 值从 5.18 变为 5.17
```

File: docs/zh/connector-v2/formats/ogg-json.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+| format                       | (none) | 是    | 指定要使用的格式，这里应该是`-json`                                                              |
+| ogg_json.ignore-parse-errors | false  | 否    | 跳过有解析错误的字段和行而不是失败。如果出现错误，字段将设置为 null                                               |
+| ogg_json.database.include    | (none) | 否    | 正则表达式，可选，通过正则匹配 Canal 记录中的`database`元字段来仅读取特定数据库变更日志行。此字符串Pattern模式与Java的Pattern兼容 |
+| ogg_json.table.include       | (none) | 否    | 正则表达式，可选，通过正则匹配 Canal 记录中的 `table` 元字段来仅读取特定表的更改日志行。此字符串Pattern模式与Java的Pattern兼容   |
+
+# 如何使用 Ogg 格式
+
+## Kafka 使用示例
+
+Ogg 为变更日志提供了统一的格式，下面是从 Oracle products 表捕获变更操作的简单示例：
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/ogg-json.md]:
```suggestion
Ogg 为变更日志提供了统一的格式，下面是从 Oracle PRODUCTS 表捕获变更操作的简单示例：
```

File: docs/zh/connector-v2/formats/ogg-json.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+    "id"
+  ],
+  "pos": "00000000000000000000143",
+  "table": "PRODUCTS"
+}
+```
+
+注：各字段含义请参考文档
+
+此 Oracle products 表有 4 列 (id, name, description 和 weight)
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/ogg-json.md]:
```suggestion
此 Oracle PRODUCTS 表有 4 列 (id, name, description 和 weight)
```

File: docs/zh/connector-v2/formats/ogg-json.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+  "current_ts": "2020-05-13 15:40:07.000000",
+  "primary_keys": [
+    "id"
+  ],
+  "pos": "00000000000000000000143",
+  "table": "PRODUCTS"
+}
+```
+
+注：各字段含义请参考文档
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/ogg-json.md]:
```suggestion
注：各字段含义请参考 [Debezium 文档](https://debezium.io/documentation/reference/2.5/connectors/oracle.html#oracle-events)
```

File: docs/zh/connector-v2/formats/ogg-json.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
+# Ogg 格式
+
+[Oracle GoldenGate](https://www.oracle.com/integration/goldengate/) (a.k.a ogg) 是一项托管服务，提供实时数据网格平台，该平台使用复制来保持数据高度可用，并支持实时分析。客户可以设计、执行和监控其数据复制和流数据处理解决方案，而无需分配或管理计算环境。 Ogg 为变更日志提供了格式架构，并支持使用 JSON 序列化消息。
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/ogg-json.md]:
```suggestion
[Oracle GoldenGate](https://www.oracle.com/integration/goldengate/) (a.k.a ogg) 是一项托管服务，提供实时数据网格平台，该平台使用复制来保持数据高度可用，并支持实时分析。客户可以设计、执行和监控其数据复制和流数据处理解决方案，而无需分配或管理计算环境。 Ogg 为变更日志提供了统一的格式结构，并支持使用 JSON 序列化消息。
```
This refers to the translation of the [Flink document](https://nightlies.apache.org/flink/flink-docs-master/zh/docs/connectors/table/formats/ogg/)

Comment (corgy-w) [reply docs/zh/connector-v2/formats/ogg-json.md]: All above have been changed, and will push later. tks！！！ fri：）

File: docs/zh/connector-v2/formats/debezium-json.md (hunk 7 above)
Comment (xxzuo) [inline docs/zh/connector-v2/formats/debezium-json.md]:
```suggestion
SeaTunnel 支持将 Debezium JSON 消息解析为 INSERT/UPDATE/DELETE 消息并导入到 seatunnel 系统中。在许多情况下，利用这个特性是非常有用的，例如:
```

File: docs/zh/connector-v2/formats/avro.md [hunk 13]
```diff
@@ -0,0 +1,111 @@
+# Avro 格式
+
+Avro是一种流行数据传输格式。现在seatunnel在kafka连接器中支持Avro格式
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/avro.md]:
```suggestion
Avro 在流式数据处理管道中非常流行。现在seatunnel在kafka连接器中支持Avro格式
```

File: docs/zh/connector-v2/formats/avro.md [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+sink {
+  Kafka {
+    bootstrap.servers = "kafkaCluster:9092"
+    topic = "test_avro_topic_fake_source"
+    format = avro
+  }
+}
+```
+
+- 用 avro 格式从 kafka 读取数据并打印到控制台的示例
```
Comment (xxzuo) [inline docs/zh/connector-v2/formats/avro.md]:
```suggestion
- 从 kafka 读取 avro 格式的数据并打印到控制台的示例
```

File: docs/zh/connector-v2/sink/Elasticsearch.md [hunk 15]
```diff
@@ -0,0 +1,218 @@
...
+
+x-pack 密码
+
+### max_retry_count [int]
+
+批量请求最大尝试大小
+
+### max_batch_size [int]
+
+批量批量文档最大大小
```
Comment (xxzuo) [inline docs/zh/connector-v2/sink/Elasticsearch.md]:
```suggestion
批次批量文档最大大小
```

File: docs/zh/connector-v2/sink/Elasticsearch.md [hunk 16]
```diff
@@ -0,0 +1,218 @@
...
+```bash
+sink {
+    Elasticsearch {
+        hosts = ["localhost:9200"]
+        index = "seatunnel-${age}"
+    }
+}
+```
+
+CDC(更改数据捕获) 事件
```
Comment (xxzuo) [inline docs/zh/connector-v2/sink/Elasticsearch.md]:
```suggestion
CDC(变更数据捕获) 事件
```
Suggest consistent translation of CDC

File: docs/zh/connector-v2/sink/Feishu.md [hunk 17]
```diff
@@ -11,7 +11,7 @@
 ## 主要特性
 
 - [ ] [精确一次](../../../en/concept/connector-v2-features.md)
```
Comment (xxzuo) [inline docs/zh/connector-v2/sink/Feishu.md]:
```suggestion
- [ ] [精确一次](../../concept/connector-v2-features.md)
```

File: docs/zh/connector-v2/sink/Jdbc.md [hunk 18]
```diff
@@ -0,0 +1,354 @@
...
+附加连接配置参数，当属性和URL具有相同参数时，优先级由<br/>
+驱动程序的具体实现确定。例如，在 MySQL 中，属性优先于 URL。
+
+### common options
+
+Sink插件常用参数，请参考 [Sink常用选项](common-options.md) 了解详情
+
+### schema_save_mode [Enum]
+
+在开启同步任务之前，针对目标面现有的表面结构选择不同的处理方案。选项介绍：
```
Comment (xxzuo) [inline docs/zh/connector-v2/sink/Jdbc.md]:
```suggestion
在开启同步任务之前，针对目标端现有的表结构选择不同的处理方案。选项介绍：
```

File: docs/zh/connector-v2/formats/avro.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
+# Avro 格式
+
+Avro是一种流行数据传输格式。现在seatunnel在kafka连接器中支持Avro格式
+
+# 怎样用
+
+## Kafka 使用示例
+
+- 使用 Avro 格式从模拟数据源生成数据并将其汇聚到 Kafka 的示例。
```
Comment (zhilinli123) [inline docs/zh/connector-v2/formats/avro.md]: 模拟随机生成数据源,并以Avro的格式 写入Kafka的实例

File: docs/zh/connector-v2/sink/Elasticsearch.md [hunk 20]
```diff
@@ -0,0 +1,218 @@
...
+        data_save_mode = "APPEND_DATA"
+    }
+}
+```
+
+## 变更日志
+
+### 2.2.0-beta 2022-09-26
+
+- 添加 Elasticsearch Sink 连接器
```
Comment (zhilinli123) [inline docs/zh/connector-v2/sink/Elasticsearch.md]: delete

File: docs/zh/connector-v2/sink/Email.md [hunk 21]
```diff
@@ -0,0 +1,87 @@
+# Email
+
+> Email 数据接收器
+
+## 描述
+
+将接收的数据作为文件发送到电子邮件
+
+已测试的电子邮件版本为 1.5.6
```
Comment (zhilinli123) [inline docs/zh/connector-v2/sink/Email.md]: ## 支持版本
 测试版本:1.5.6(供参考)

File: docs/zh/connector-v2/sink/Jdbc.md [hunk 22]
```diff
@@ -0,0 +1,353 @@
...
+    jdbc {
+        url = "jdbc:mysql://localhost:3306"
+        driver = "com.mysql.cj.jdbc.Driver"
+        user = "root"
+        password = "123456"
+        
+        database = "sink_database"
+        table = "sink_table"
+        primary_keys = ["key1", "key2", ...]
+        schema_save_mode = "CREATE_SCHEMA_WHEN_NOT_EXIST"
```
Comment (zhilinli123) [inline docs/zh/connector-v2/sink/Jdbc.md:307]: 中文描述这个功能具体做了什么？英文文档同上！

File: docs/zh/connector-v2/sink/Jdbc.md [hunk 23]
```diff
@@ -0,0 +1,353 @@
...
+        user = "root"
+        password = "123456"
+        compatible_mode="postgresLow"
+        database = "sink_database"
+        table = "sink_table"
+        support_upsert_by_query_primary_key_exist = true
+        generate_sink_sql = true
+        primary_keys = ["key1", "key2", ...]
+    }
+}
```
Comment (zhilinli123) [inline docs/zh/connector-v2/sink/Jdbc.md:331]: 简单描述这个case是用来做什么的

File: docs/zh/connector-v2/sink/Jdbc.md [hunk 24]
```diff
@@ -0,0 +1,353 @@
...
+
+```
+
+## 变更日志
+
+### 2.2.0-beta 2022-09-26
+
+- 添加控制台接收器连接器
```
Comment (zhilinli123) [inline docs/zh/connector-v2/sink/Jdbc.md]: Delete

**Comment by corgy-w** [comment]
@zhilinli123  @EricJoy2048 E I have completed PR self-review, please check , if there is any problem, I will fix it immediately. Tks！

**Review by zhilinli123 — APPROVED** [review]
LGTM,thanks contribution,PTAL: @hailin0 @EricJoy2048

**Comment by corgy-w** [comment]
Is there any pending point in this pr ？PTAL： @hailin0 @EricJoy2048 tks1

File: docs/zh/connector-v2/formats/ogg-json.md [hunk 25]
```diff
@@ -0,0 +1,93 @@
...
+  ],
+  "pos": "00000000000000000000143",
+  "table": "PRODUCTS"
+}
+```
+
+注：各字段含义请参考 [Debezium 文档](https://debezium.io/documentation/reference/2.5/connectors/oracle.html#oracle-events)注：各字段含义请参考 [Debezium 文档](https://debezium.io/documentation/reference/2.5/connectors/oracle.html#oracle-events)
+
+此 Oracle PRODUCTS 表有 4 列 (id, name, description 和 weight)
+上面的 JSON 消息是 products 表上的更新更改事件，其中 id = 111 的行的权重值从 5.18 更改为 5.15。
```
Comment (EricJoy2048) [inline docs/zh/connector-v2/formats/ogg-json.md]:
```suggestion
上面的 JSON 消息是 products 表上的更新更改事件，其中 id = 111 的行的字段`weight`的值从 5.18 更改为 5.15。
```

File: docs/zh/connector-v2/formats/ogg-json.md [hunk 26]
```diff
@@ -0,0 +1,93 @@
...
+假设此表的 binlog 的消息已经同步到 Kafka topic，那么我们可以使用下面的 SeaTunnel 示例来消费这个主题并体现变更事件。
```
Comment (EricJoy2048) [inline docs/zh/connector-v2/formats/ogg-json.md]:
```suggestion
假设此表的 binlog 的消息已经同步到 Kafka topic，那么我们可以使用下面的 SeaTunnel 示例来消费这个topic并体现变更事件。
```

File: docs/zh/connector-v2/sink/Elasticsearch.md [hunk 27]
```diff
@@ -0,0 +1,218 @@
...
+### common options
+
+Sink插件常用参数，请参考 [Sink常用选项](common-options.md) 了解详情
+
+### schema_save_mode
+
+在启动同步任务之前，针对目标侧已有的表结构选择不同的处理方案<br/>
+选项介绍：<br/>
+`RECREATE_SCHEMA` ：当表不存在时会创建，当表已存在时会删除并重建<br/>
+`CREATE_SCHEMA_WHEN_NOT_EXIST` ：当表不存在时会创建，当表已存在时则跳过<br/>
```
Comment (EricJoy2048) [inline docs/zh/connector-v2/sink/Elasticsearch.md]:
```suggestion
`CREATE_SCHEMA_WHEN_NOT_EXIST` ：当表不存在时会创建，当表已存在时则跳过创建<br/>
```

File: docs/zh/connector-v2/sink/Kafka.md [hunk 28]
```diff
@@ -0,0 +1,196 @@
...
+| name | age |     data      |
+|------|-----|---------------|
+| Jack | 16  | data-example1 |
+| Mary | 23  | data-example2 |
+
+如果 `${name}` 设置为 topic。因此，第一行发送到 Jack topic，第二行发送到 Mary topic。
+
+### 语义
+
+在 EXACTLY_ONCE 中，生产者将在 Kafka 事务中写入所有消息，这些消息将在检查点上提交给 Kafka
```
Comment (EricJoy2048) [inline docs/zh/connector-v2/sink/Kafka.md]:
```suggestion
在 EXACTLY_ONCE 中，生产者将在 Kafka 事务中写入所有消息，这些消息将在检查点上提交给 Kafka，该模式下能保证数据精确写入kafka一次，即使任务失败重试也不会出现数据重复和丢失
```

File: docs/zh/connector-v2/sink/Kafka.md [hunk 29]
```diff
@@ -0,0 +1,196 @@
...
+在 AT_LEAST_ONCE 中，生产者将等待 Kafka 缓冲区中所有未完成的消息在检查点上被 Kafka 生产者确认
```
Comment (EricJoy2048) [inline docs/zh/connector-v2/sink/Kafka.md]:
```suggestion
在 AT_LEAST_ONCE 中，生产者将等待 Kafka 缓冲区中所有未完成的消息在检查点上被 Kafka 生产者确认，该模式下能保证数据至少写入kafka一次，即使任务失败重试也不会出现数据丢失，但可能有会发生数据重复。
```

File: docs/zh/connector-v2/sink/Kafka.md [hunk 30]
```diff
@@ -0,0 +1,196 @@
...
+NON 不提供任何保证：如果 Kafka 代理出现问题，消息可能会丢失，并且消息可能会重复
```
Comment (EricJoy2048) [inline docs/zh/connector-v2/sink/Kafka.md]:
```suggestion
NON 不提供任何保证：如果 Kafka 代理出现问题，消息可能会丢失，并且消息可能会重复，该模式下，任务失败重试可能会产生数据丢失或重复。
```

**Comment by corgy-w** [comment]
@EricJoy2048  Sorry causing you to check so many problems again, they all be complete , TKS！

**Comment by corgy-w** [comment]
> @EricJoy2048 Sorry causing you to check so many problems again, they all be complete , TKS！

@EricJoy2048 hi If there is still a problem, I will fix the first time.

**Review by EricJoy2048 — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by hailin0 — APPROVED** [review]

**Pull request merged**